test/tools/yulopti.cpp

```diff
@@ -265,10 +369,11 @@ int main(int argc, char** argv)
 		po::options_description options(
 			R"(yulopti, yul optimizer exploration tool.
 	Usage: yulopti [Options] <file>
-	Reads <file> as yul code and applies optimizer steps to it,
+	Reads <file> containing a yul object and applies optimizer steps to it,
 	interactively read from stdin.
 	In non-interactive mode a list of steps has to be provided.
 	If <file> is -, yul code is read from stdin and run non-interactively.
+	If <object> is provided then only the object matching the given path will be read.
```

Comment:
Can you explain what `<object>` is?

Reply:
I've changed this to `An <object> flag may be provided, specifying a dotted path to an object in the input.`. Hope that's better.